db/models/articles-model.js

```diff
@@ -97,6 +97,47 @@ exports.createArticleComment = (article_id, username, body) => {
         }  
     })
 }
-// exports.updateArticleById =() => {
-
-// }
\ No newline at end of file
+exports.updateArticleById =(votes_inc, article_id) => {  
+    if(isNaN(votes_inc)) {
```

Comment:
We likely don't need a custom error handler here, as a general rule always allow psql to throw an error if it's going to and only write a custom error handler if psql doesn't throw an error. Try seeing what happens if you pass the incorrect input to the query.

Reply:
done